lib/que/locker.rb

```diff
@@ -12,9 +14,27 @@ module Que
   # For more information, see the 'Predicate Specificity' chapter of:
   # https://brandur.org/postgres-queues
   class Locker
-    def initialize(queue:, cursor_expiry:, metrics:)
+    ExistsTotal = Prometheus::Client.registry.counter(
+      :que_locker_exists_total, "Counter of attempts to check job existence before locking",
+    )
+    ExistsSecondsTotal = Prometheus::Client.registry.counter(
+      :que_locker_exists_seconds_total, "Seconds spent checking job exists before locking",
+    )
+    UnlockTotal = Prometheus::Client.registry.counter(
+      :que_locker_unlock_total, "Counter of attempts to unlock advisory job locks",
```

Comment:
`Counter of attempts to unlock job advisory locks` (switch the word order around - it's the lock that's advisory, not the job)